build.gradle

```diff
@@ -23,6 +23,20 @@ plugins {
   id 'io.codearte.nexus-staging' version '0.11.0'
 }
 
+subprojects {
+  ext {
+    versions = [
+      'assertj_core': '3.11.1',
+      'commons_compress': '1.21',
+      'jackson_databind': '2.15.1',
+      'junit':          '4.12',
+      'mockito':        '2.5.7',
+      'slf4j':          '1.7.36',
+      'wiremock_jre':   '2.35.0'
```

Comment:
Minor: Either align all values or none.

Reply:
Decided to not align. See [8ec34cc](https://github.com/metafacture/metafacture-core/pull/489/commits/8ec34cc2b413cdfa0b3315af8344f7ab8af85849).